changelogs/unreleased/bump-libcurl-to-8.3.0.md

```diff
@@ -1,3 +1,3 @@
-## feature/build
+## bugfix/build
 
-* Updated libcurl to version 8.3.0.
```

Comment:
Let's look into the words.

The previous changelog entry states the following: we updated a build time dependency. Feature? Not exactly, rather a change in an existing functionality.

The new entry states that we fixed a bug (`## bugfix`) in the build machinery (`/build`). So, there was a build failure? Or incorrect executable file? Seems, no. There is no bug in the build machinery.

Next, what is the bug? libcurl-7.87.0 is not a bug, libcurl 8.3.0 is not a fix. If we want to tell about a bugfix, we should include some problem statement. In fact, the issue states a problem in the built-in http client and I think that we should tell about it here.

How exactly the problem is fixed (by a change in the build machinery) is on the deeper level.

Reply:
fixed

Reply:
I would write `## bugfix/lua/http client`.

Reply:
It appears we have another entry in our list:

git show 83ddadb46ef110948960209cb9197b85372109fb
```
diff --git a/changelogs/unreleased/bump-libcurl-to-8.3.0.md b/changelogs/unreleased/bump-libcurl-to-8.3.0.md
new file mode 100644
index 0000000000..ee7d9e3b45
--- /dev/null
+++ b/changelogs/unreleased/bump-libcurl-to-8.3.0.md
@@ -0,0 +1,3 @@
+## feature/build
+
+* Updated libcurl to version 8.3.0.
diff --git a/changelogs/unreleased/gh-9086-fix-streaming-connection-stuck.md b/changelogs/unreleased/gh-9086-fix-streaming-connection-stuck.md
new file mode 100644
index 0000000000..34a10bb183
--- /dev/null
+++ b/changelogs/unreleased/gh-9086-fix-streaming-connection-stuck.md
@@ -0,0 +1,4 @@
+## bugfix/http
+
+* Fixed a streaming connection stuck if etcd is stopped
+  unexpectedly (gh-9086).
```

I believe we have to merge them in one, since they're about one fix - perhaps change the `bugfix/http` for `bugfix/lua/http client` according to your request.